csfieldguide/curriculum_guides/content/en/apcsp/sections/global-impact.md

```diff
@@ -1,25 +1,35 @@
 # Global Impact
 
 ## Overview
+
 - EU 7.1 Computing enhances communication, interaction, and cognition.
 - EU 7.2 Computing enables innovation in nearly every field.
 - EU 7.3 Computing has a global affect — both beneficial and harmful — on people and society.
 - EU 7.4 Computing innovations influence and are influenced by the economic, social, and cultural contexts in which they are designed and used.
 - EU 7.5 An investigative process is aided by effective organization and selection of resources. Appropriate technologies and tools facilitate the accessing of information and enable the ability to evaluate the credibility of sources.
 
-
 ## Reading from the Computer Science Field Guide
 
+- CSFG content is not planned for these.
+
 ## Learning objectives
-The above chapter readings include specific knowledge for EK's marked in bold. Work to include unmarked learning objectives in the CS Field Guide is currently in progress.
+
+The above chapter readings include specific knowledge for EKs marked in bold. Work to include unmarked learning objectives in the CS Field Guide is currently in progress.
```

Comment:
This statement contradicts the one above. Also none of the EK's are marked in bold anyway, so if content is not planned for these then we should remove this statement entirely.

Reply:
None of them are marked on bold *because* of the statement above, but I get your point. Will remove